README.md

```diff
@@ -1,2 +1,18 @@
-# Quick start
-`make vendor`
\ No newline at end of file
+# Assured Ledger
+
+[![codecov](https://codecov.io/gh/insolar/assured-ledger/branch/master/graph/badge.svg)](https://codecov.io/gh/insolar/assured-ledger)
+
+Assured Ledger is a byzantine fault tolerant distributed application server.
+
+This project is currently under development. It's the next generation of [Insolar Platform](https://github.com/insolar/insolar).
+
+## Quick start
+
+How to build the project:
+
+```
+cd ledger-core/v2
+make vendor pre-build build
+````
+
+For more details please see .github/workflows/common.yaml.
```

Comment:
what details supposed to be there? Why we want somebody to go checl our github actions workflow in yaml? 🐙

Reply:
Good point.